src/SourceBuild/content/eng/tools/tasks/Microsoft.DotNet.SourceBuild.Tasks.XPlat/UsageReport/WriteUsageBurndownData.cs

```diff
@@ -85,11 +97,9 @@ public override bool Execute()
         /// <param name="relativeFilePath">The relative path (from the git root) to the file.</param>
         /// <param name="commitSha">The commit sha for the version of the file to get.</param>
         /// <returns>The contents of the specified file.</returns>
-        private string GetFileContents(string relativeFilePath, string commitSha)
+        private async Task<string> GetFileContentsAsync(string relativeFilePath, string commitSha)
         {
-            WebClient client = new WebClient();
-            var xmlString = client.DownloadString($"https://raw.githubusercontent.com/dotnet/source-build/{commitSha}/{relativeFilePath.Replace('\\', '/')}");
-            return xmlString;
+            return await client.GetStringAsync($"https://raw.githubusercontent.com/dotnet/source-build/{commitSha}/{relativeFilePath.Replace('\\', '/')}");
         }
```

Comment:
This can be simplified:

```c#
private Task<string> GetFileContentsAsync(string relativeFilePath, string commitSha) =>
    client.GetStringAsync($"https://raw.githubusercontent.com/dotnet/source-build/{commitSha}/{relativeFilePath.Replace('\\', '/')}");
```

Reply:
Addressed in commit: [Simplify GetFileContentsAsync by using lamda](https://github.com/dotnet/installer/pull/17128/commits/dbd2384d43cb240f9fade27003576c78e4912d5a)